examples/create-react-app/README.md

```diff
@@ -0,0 +1,10 @@
+## Example: create-react-app
+
+This shows the usage of CMS design system components in a [create-react-app](https://create-react-app.dev/docs/getting-started/) project.
+
+Refer to [install with NPM]({{root}}/startup/installation/) for instruction on installing the design system as a dependency package. This example includes a basic `.env` and `index.scss` to reference the dist folder.
```

Comment:
Could we expand on this? What does "reference the dist folder" mean? Maybe we can say the `.env` is for a basic create-react-app configuration with the design system.